inference/core/active_learning/README.md

```diff
@@ -0,0 +1,269 @@
+video streams and the HTTP API, both on-premises and within the Roboflow platform.
+* During runtime, as predictions are generated, images and model predictions (treated as initial annotations) are 
+dynamically collected and submitted in batches into user project. These batches are then ready for labeling within the 
+Roboflow platform.
+
+To decide which datapoints should be collected - Active Learning components use sampling strategies that can be 
+defined by users.
+
+## Active Learning configuration
+Active learning configuration contains **sampling strategies** and other fields influencing the feature behaviour.
+
+#### Configuration options
+* `enabled` - boolean flag to enable / disable the configuration (required) - `{"enabled": false}` is minimal valid config
+* `max_image_size` - two element list with positive integers (height, width) enforcing down-sizing (with aspect-ratio
+preservation) of images before submission into Roboflow platform (optional)
+* `jpeg_compression_level` - integer value in range [1, 100]  representing compression level of submitted images 
+(optional, defaults to `95`)
+* `persist_predictions` - binary flag to decide if predictions should be collected along with images (required if `enabled`)
+* `sampling_strategies` - list of sampling strategies (non-empty list required if `enabled`)
+* `batching_strategy` - configuration of labeling batches creation - details below (required if `enabled`)
+* `tags` - list of tags (each contains 1-64 characters from range `a-z, A-Z, 0-9, and -_:/.[]<>{}@`) (optional)
+
+#### Batching strategy
+`batching_strategy` field holds a dictionary with the following configuration options:
+* `batches_name_prefix` - string representing the prefix of batches names created by Active Learning (required)
+* `recreation_interval` - one of value `["never", "daily", "weekly", "monthly"]` - representing the interval which
+is to be used to create separate batches - thanks to that - user can control the flow of labeling batches in time
+(required)
+* `max_batch_images` - positive integer representing maximum size of batch (applied on top of any strategy limits)
+to prevent too much data to be collected (optional)
+
+#### Example configuration
+```json
+{
+    "enabled": true,
+    "max_image_size": [1200, 1200],
+    "jpeg_compression_level": 75,
+    "persist_predictions": true,
+    "sampling_strategies": [
+      {
+        "name": "default_strategy",
+        "type": "random",
+        "traffic_percentage": 0.1
+      }
+    ],
+    "batching_strategy": {
+      "batches_name_prefix": "al_batch",
+      "recreation_interval": "daily"
+    }
+}
+```
+
+## Sampling strategies
+
+Every user-defined strategy must possess a unique name within a given configuration. In addition to the name, 
+each strategy is distinguished by its specified type. Various types of strategies are available for utilization.
+
```

Comment:
Can you make a list of all the strategies here that links to all of the headings? The links should be `#random-sampling-strategy`, etc.